libs/flux/build.go

```diff
@@ -450,3 +458,20 @@ func removeEnvVar(env []string, key string) []string {
 	}
 	return env
 }
+
+func setEnvVar(env []string, key string, value string) []string {
+	found := false
```

Comment:
Instead of using a boolean here, you can return on line 470 instead of `break`. So something like:
```
env[i] = envString
return env
```

Reply:
I thought about that, but wasn't sure if I was leaving something readable there. I thought my intentions would be clearer with a boolean.